plugins/module_utils/utils.py

```diff
@@ -145,3 +145,29 @@ def recursive_delete(data, path, keys):
 
         for key in remove_data:
             recursive_delete(existing_data, update_path, key if isinstance(key, tuple) else (key,))
+
+
+def format_list_dict(list_dict, keys_map):
+    """
+    Convert an Python list of dictionaries into its equivalent NDO API format.
+    All keys must be defined in the keys map even if no conversion is needed for some keys.
+
+    :param list_dict: The Python list of dictionaries to format. -> List[Dict]
+    :param keys_map: the mapping from the Ansible argument's keys to NDO API keys. Can also include the map between values -> Dict
+    :return: The formatted dictionary. -> Dict
+    """
+    if isinstance(list_dict, list):
```

Comment:
we actually don't need to do that cause it's a function to format attribute inputs that are of type List of dict and there are three type of scenarios in this case: 
1. it's list of dict so it's passing through correctly.
2. the attribute is None (it's not been set up in the Ansible playbook). Therefore, the function will return None which what we want in this case.
3. the attribute is of another type, the module will fail way before it's passing through the function since it's not of the correct type.

Reply:
I added them just in case, but should not be needed.